packages/yourloops/test/integration/pages/home-page/hcp-home-page.spec.tsx

```diff
@@ -216,4 +217,58 @@ describe('HCP home page', () => {
 
     expect(invitePatientButton).toBeEnabled()
   })
+
+  it('should check that the team creation button is valid if all fields are complete', async () => {
+    await act(async () => {
+      render(getHomePage())
```

Comment:
```suggestion
      renderPage('/')
```

renderPage import: `import { renderPage } from '../../utils/render'`